themes/socialblue/src/Plugin/Preprocess/Page.php

```diff
@@ -25,11 +25,21 @@ public function preprocess(array &$variables, $hook, array $info) {
       $route_match = \Drupal::routeMatch();
       if ($route_match->getParameter('user') && $route_match->getRouteName() !== 'entity.profile.type.user_profile_form') {
         $variables['content_attributes']->addClass('sidebar-left', 'content-merged--sky');
+
+        // If page has title.
```

Comment:
@agami4 Please add more clear comment, which explains why you need to hide a title here. See example from socialbase:

```
        // If there is a title and node type is excluded remove class.
        if (in_array($node->bundle(), $page_to_exclude, TRUE)) {
          $attributes->removeClass('with-title-region');
          $variables['display_page_title'] = FALSE;
        }
```
And I think this condition is not neded here `if ($variables['page']['title']) {`
But should we also add line `$attributes->removeClass('with-title-region'); ` here?